readme.md

```diff
@@ -109,6 +109,7 @@
 
 #### Theory
 
+- 1996: Lawall and Mairson, "Optimality and Inefficiency: What Isn't a Cost Model of the Lambda Calculus?" [url](https://dl.acm.org/doi/10.1145/232629.232639)
```

Comment:
This is already in the "efficiency" section (great paper!)

Is there a reason to duplicate it? If yes I would move the second instance to "optimality"